framework_tvb/tvb/core/services/simulator_serializer.py

```diff
@@ -94,7 +93,7 @@ def deserialize_simulator(simulator_gid, storage_path):
             with CortexH5(cortex_path) as cortex_h5:
                 simulator_in.surface.local_connectivity = cortex_h5.local_connectivity.load()
                 simulator_in.surface.region_mapping_data = cortex_h5.region_mapping_data.load()
-                rm_index = dao.get_datatype_by_gid(simulator_in.surface.region_mapping_data.hex)
-                simulator_in.surface.fk_surface_gid = uuid.UUID(rm_index.fk_surface_gid)
```

Comment:
Is this correct ?

Reply:
This would probably break the Simulator GUI on load/copy operations, but it does not affect the simulation run and it was a quick way to check that surface simulations work. 
Will be treated in a better way together with the ProjectionMonitor deserialization done above, which also queries the DB.